vm/src/obj/objsuper.rs

```diff
@@ -80,8 +115,6 @@ fn super_init(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
         ));
     }
 
-    // TODO: how to store those types?
```

Comment:
This TODO can now be solved by using new-style classes as can be seen in objint.rs for example. Implement a struct named `struct PySuper { PyObjectRef obj }` and use this struct as a payload. Note that the init method might need to be changed to a super_new method.